lms/static/sass/shared/_footer-edx.scss

```diff
@@ -4,20 +4,102 @@
 @import 'neat/neat'; // lib - Neat
 
 
+@import '../../../../node_modules/bootstrap/scss/functions';
```

Comment:
These can just be:

    @import 'bootstrap/scss/functions';

as `node_modules` is on the Sass loader path.